test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1590,28 +1600,17 @@ class test_ucp_sockaddr_protocols_err : public test_ucp_sockaddr_protocols {
                                                         variants & RECV_STOP);
     }
 
-    void cleanup() {
-        test_ucp_sockaddr_protocols::cleanup();
-    }
-
-    static void err_handler_cb(void *arg, ucp_ep_h ep, ucs_status_t status) {
```

Comment:
why removed?

Reply:
we close receiver/sender EPs after making an attempt to do an extra send operation
so, we don't need special `err_handler_cb()` which do closes EPs

Reply:
ok